### PR TITLE
feat: Reusable Button component

### PR DESCRIPTION
**Pull request by rlaferla**

### CODE REVIEW

File: src/components/Button.svelte [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+  export let title: string = '';
+  export let action = () => {};
+</script>
+
+<div class="flex w-350 justify-between">
```
Comment (claireolmstead) [inline src/components/Button.svelte:6]: should not need this styling. seems like leftover code. additionally, if you find this to be worth it, look into adding the svelte version of `{...props}` - a prop that can accept any standard HTML button prop.

File: src/components/Button.svelte [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+  export let title: string = '';
+  export let action = () => {};
```
Comment (claireolmstead) [inline src/components/Button.svelte:3]: nit: rename to onClick so that we don't have to remember a different name.

**Review by claireolmstead — COMMENTED** [review]
Thank you for prioritizing reusability.

**Review by mattheworris — APPROVED** [review]
🚢 it!

**Pull request merged**